easybuild/tools/utilities.py

```diff
@@ -68,14 +68,14 @@ def quote_str(x):
     """
 
     if isinstance(x, basestring):
-        if "'" in x and '"' in x:
+        if '\n' in x or ("'" in x and '"' in x):
             return '"""%s"""' % x
         elif '"' in x:
             return "'%s'" % x
         else:
             return '"%s"' % x
     else:
-        return x
+	return x
```

Comment:
no tabs please, see http://legacy.python.org/dev/peps/pep-0008/#tabs-or-spaces

add this to your `~/.vimrc`:
```
:set expandtab tabstop=4 shiftwidth=4
```